.github/workflows/publish.yml

```diff
@@ -0,0 +1,90 @@
+# This workflow will upload a Python Package to Release asset
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Create Release
+
+on:
+  push:
+    branches:
+      - release/*
+
+# Needed to create release and upload assets
+permissions:
+  contents: write
+
+jobs:
+  release:
+    # Tags and releases the branch
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Extract branch info
+        shell: bash
+        # The tag is the branch name with the prefix removed.
+        # for example: release/v1.0.0 -> v1.0.0
+        run: |
+          release_tag=$(echo "${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" | sed -e 's/.*\///g')
+          echo "release_tag=$release_tag" >> $GITHUB_ENV
+        id: extract_branch
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.release_tag }}
+          release_name: ${{ env.release_tag }}
+
+  wheel:
+    name: Build Wheel
+    runs-on: ${{ matrix.os }}
+    needs: release
+    
+    strategy:
+      fail-fast: false
+      matrix:
+          os: ['ubuntu-20.04']
+          python-version: ['3.8', '3.9', '3.10', '3.11']
+          cuda-version: ['11.8'] # Github runner can't build anything older than 11.8
```

Comment:
Awesome! BTW, could you provide a link to any document about this restriction?

Reply:
I didnt find a document about this restriction.
When I tried using Cuda 11.7 and building the project I received errors about incompatible GPU (even when I tried skipping the runtime error thrown in setup.py).
Might be possible to make 11.7 work, I didn't focus on that too much.

Reply:
The python versions simply match the ones supported in torch's repo. https://download.pytorch.org/whl/cu118/torch_stable.html

Reply:
Got it. Thanks for the explanation. I think this is totally ok.